test/langPickerTest.js

```diff
@@ -147,6 +147,45 @@ describe('LanguagePicker', () => {
       ],
       expected: 'en value',
     },
+    /* Script overrides */
+    {
+      msg: 'Fallback to language script be-tarask -> Cyrl',
+      langCode: 'be-tarask',
+      config: {
+        languageMap: {
+          foo: 'bar',
+          other: 'languages',
+          that: 'dont',
+          matter: 'at all',
+        },
+      },
+      values: [
+        { en: 'en value' },
+        { he: 'he value' },
+        { es: 'es value' },
+        { Cyrl: 'Cyrl value' },
```

Comment:
here you should have  `ru: 'ru value'` instead of Cyrl. There is no such thing as "Cyrl" in OSM db.